### PR TITLE
[FIX] udes_open,edi_notifier: Added hostname to success, failed and n…

### PR DESCRIPTION
**Pull request by nickcosta85**
…ot received notifers

Story/15274

Signed-off-by: Nick Costa <nicholas.costa@unipart.io>

### CODE REVIEW

File: addons/edi_notifier/models/email_notifier.py [hunk 1]
```diff
@@ -18,6 +20,8 @@ def _should_notify(self, notifier, event_type, rec):
     @api.multi
     def _notify(self, notifier, event_type, recs):
         base_url = self.env['ir.config_parameter'].sudo().get_param('web.base.url')
+        hostname = socket.gethostname()
+        print("hostname ======", hostname)
```
Comment (joe-gb) [inline addons/edi_notifier/models/email_notifier.py:24]: remove print statement

File: addons/edi_notifier/models/email_notifier.py [hunk 2]
```diff
@@ -38,8 +42,9 @@ def _notify(self, notifier, event_type, recs):
             if attachments:
                 email_values = {'attachment_ids': attachments.mapped('id')}
             if base_url:
-                template = template.with_context(instance_url=base_url)
+                template = template.with_context(instance_url=base_url, hostname=hostname)
             template.send_mail(rec.id, force_send=True, email_values=email_values)
+
```
Comment (joe-gb) [inline addons/edi_notifier/models/email_notifier.py:47]: don't think this newline is needed

**Review by joe-gb — CHANGES_REQUESTED** [review]

File: addons/edi_notifier/models/email_notifier.py (hunk 1 above)
Comment (nickcosta85) [reply addons/edi_notifier/models/email_notifier.py]: my mistake, I have removed this now.

File: addons/edi_notifier/models/email_notifier.py (hunk 2 above)
Comment (nickcosta85) [reply addons/edi_notifier/models/email_notifier.py:46]: my mistake, I have removed this now also.

**Comment by apatrick88** [comment]
Can you tidy up the commit message and squash into one commit before merging please? E.G.:

[FIX] edi_notifier: Brief blah

More detailed blah

Story/blah

**Review by adamrpatrick — APPROVED** [review]

**Review by joe-gb — APPROVED** [review]

**Pull request merged**